packages/web-react/src/components/Navigation/useNavigationLinkProps.ts

```diff
@@ -0,0 +1,26 @@
+import { SpiritNavigationLinkProps } from '../../types';
+
+export type UseNavigationLinkProps = Partial<SpiritNavigationLinkProps> & {
+  'aria-current'?: string;
+};
+
+export type UseNavigationLinkReturn = {
+  navigationLinkProps: UseNavigationLinkProps;
+};
+
+export const useNavigationLinkProps = (props: UseNavigationLinkProps): UseNavigationLinkReturn => {
+  const { elementType = 'a', isDisabled, href, target, rel, ariaLabel } = props;
+
+  const additionalProps: Partial<SpiritNavigationLinkProps> = {
+    href: elementType === 'a' && !isDisabled ? href : undefined,
+    target: elementType === 'a' && !isDisabled ? target : undefined,
+    rel: elementType === 'a' ? rel : undefined,
+    'aria-label': ariaLabel,
```

Comment:
the same for this. do we need to define this?

Reply:
You're right. We do not need it. Removed.